### PR TITLE
Update some links to https

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Apache jclouds&reg;
 
-This repository supports the GitHub Pages site for jclouds. See and read more at [http://www.jclouds.org](http://www.jclouds.org).
+This repository supports the GitHub Pages site for jclouds. See and read more at [https://www.jclouds.org](https://www.jclouds.org).
```
Comment (gaul) [inline README.md:3]: This link doesn't work.  Update to https://jclouds.apache.org/ .

**Review by gaul — CHANGES_REQUESTED** [review]
I think this is probably fine to merge after addressing the comment but did you test these URLs in any way?

**Comment by raboof** [comment]
> did you test these URLs in any way?

I tested a bunch of them by (manually) following the updated link and making sure it loaded - indeed that's not foolproof, as I clearly missed the one in the README.

**Comment by gaul** [comment]
Thank you for your contribution @raboof!

**Comment by gaul** [comment]
@nacx I cannot deploy the site (actually never have been able), perhaps due to running Fedora with its podman container setup.  I fixed a few things in the deploy scripts but could you try pushing this?

**Comment by gaul** [comment]
One hint is that my image has unexpected files instead of the exposed volume:

```
/:
/:/jclouds-site
/.dockerenv
/jclouds-site
```

**Comment by gaul** [comment]
#23 addresses the above.  Fedora needs to use https://github.com/gaul/jclouds-site/pull/new/podman to publish the site.

**Pull request merged**